framework/packages/abstract-core/src/adapter.rs

```diff
@@ -121,6 +124,17 @@ pub enum BaseQueryMsg {
     /// Returns [`AuthorizedAddressesResponse`].
     #[returns(AuthorizedAddressesResponse)]
     AuthorizedAddresses { proxy_address: String },
+    /// Returns module data
+    #[returns(AdapterModuleDataResponse)]
+    BaseModuleData {},
```

Comment:
Might consider naming this `ModuleData` instead - what value does `Base` add?

Reply:
We want to avoid name collision for schema generation, for example if we will have some adapter, which will have `ModuleData` query it won't be able to generate schema. IMO `ModuleData` is pretty generic name, wdyt?

Reply:
I think `ModuleData` is quite specific so agree with Adair on this one :)